initialize.go

```diff
@@ -14,6 +14,7 @@ const (
 	defaultPlatformURL             = "http://localhost:9000"
 	defaultMessagingURL            = "localhost:1883"
 	defaultAdapterConfigCollection = "adapter_config"
+	defaultFatalOnDisconnect       = "false"
 )
```

Comment:
I'm thinking the default value should be true, since this is the functionality a large number of the adapters already use.

Reply:
@ryanmccluretx expressed concern on my original implementation (no flag), so I added this, with false as default.